### ISSUE TITLE
Configurable number of columns (Layout)

### ISSUE THREAD

**Issue by keplerk** [PD75/practical-startpage#77]
Any chance to be able to hide sections ?

**Comment by PD75** [comment]
@keplerk , just to clarify with section you mean the red columns? (I borrowed the name from [Semantic-UI](https://github.com/Semantic-Org/Semantic-UI)). the purple are the actual widgets showing data.

![v0 2 2 - std res](https://cloud.githubusercontent.com/assets/10294795/14449298/c220e0d8-0071-11e6-8f89-24e996f88d1b.png)

**Comment by keplerk** [comment]
Yes, the columns (in this case taking as example the red rounded)

**Comment by PD75** [comment]
@keplerk , Yes I would like this as well...a bit tricky, so I did not want to start it unless someone asked for it; so thank you!!! I have given it a lot of thought over quite some time, so let me try to explain my idea.
- User can set the number of columns and width (with in reason)
- Further it will be possible to decide what number of columns for which screen resolution for those with multiple screens/computers
- Also change internal layout, e.g. number of columns inside

Sounds OK?

**Comment by keplerk** [comment]
Sounds good. ill keep following the development of this add on. :+1:

**Comment by atheistics** [comment]
That would be a nice feature...i find myself using bookmarks and history most often....not so much the quicklinks.

**Comment by PD75** [comment]
@atheistics you can already now remove and add the widgets, so just remove the quicklinks from bottom right corner options menu.
